verkle-trie/benches/benchmarks/proof_ipa.rs

```diff
@@ -0,0 +1,96 @@
+use ark_ff::{BigInteger256, UniformRand};
+use banderwagon::Fr;
+use criterion::{black_box, criterion_group, BenchmarkId, Criterion};
+use ipa_multipoint::{
+    crs::CRS,
+    lagrange_basis::{LagrangeBasis, PrecomputedWeights},
+    multiproof::{MultiPoint, ProverQuery},
+    transcript::Transcript,
+};
+use rand::Rng;
+use rayon::prelude::IntoParallelIterator;
+use rayon::prelude::*;
+
+const VECTOR_WIDTH :usize = 256;
+
+fn multiproof(c: &mut Criterion) {
+    let mut group = c.benchmark_group("multiproof");
+
+    let crs = CRS::new(VECTOR_WIDTH, b"bench");
+    let precomp = PrecomputedWeights::new(VECTOR_WIDTH);
+
+    let prover_queries = (0..16_000)
+        .into_par_iter()
+        .map(|_| {
+            LagrangeBasis::new(
+                (0..VECTOR_WIDTH)
+                    .map(|_| Fr::rand(&mut rand::thread_rng()))
+                    .collect::<Vec<_>>(),
+            )
+        })
+        .map(|poly| {
+            let point = rand::thread_rng().gen_range(0..VECTOR_WIDTH);
+            let result = poly.evaluate_in_domain(point);
+            ProverQuery {
+                commitment: crs.commit_lagrange_poly(&poly),
+                poly,
+                point,
+                result,
+            }
+        })
+        .collect::<Vec<_>>();
+
+    for num_keys in [1, 1_000, 2_000, 4_000, 8_000, 16_000] {
```

Comment:
Creating a similar setup for [our Go benchmarks](https://github.com/jsign/vkt-proof-bench).